src/labelle/lib/constants.py

```diff
@@ -95,3 +95,9 @@ class Align(str, Enum):
     LEFT = "left"
     CENTER = "center"
     RIGHT = "right"
+
+
+class Justify(str, Enum):
+    LEFT = "left"
+    CENTER = "center"
+    RIGHT = "right"
```

Comment:
What's the difference with `Align`? Looks like unnecessary duplication. If there's a reason, let's explain it so that the code is readable.

Reply:
I did not want to have `Align` class used for justify field, and vice-versa, and I did not find any other name which will be neither Align nor Justify.
I opted for duplicity in values, but I think the con is that we do not confuse these two fields. There's already some mental load on the user to understand what align means, and what justify means, so I didn't want to add to that by sharing the same struct.

Reply:
I created a single `Direction` enum.